src/TraceEvent/Stacks/CallTree.cs

```diff
@@ -50,10 +50,10 @@ public StackSource StackSource
 
                 // TODO FIX NOW I turned of Parallelism because there were signs of races and 
```

Comment:
Please move the TODO comment to the DIableParallelism property (Where the default is set to true).